app/templates/administrative-units/administrative-unit/index.hbs

```diff
@@ -162,12 +176,28 @@
                       <p><strong>Naam eredienst</strong></p>
                     </div>
                     <div class="au-o-grid__item au-u-3-4">
-                      <AuLabel for="naam-eredienst" class="au-u-hidden-visually">Naam eredienst input</AuLabel>
-                      <AuInput id="naam-eredienst"  @width="block" type="text" value={{@model.name}}/>
+                      <AuLabel
+                        for="naam-eredienst"
+                        class="au-u-hidden-visually"
+                      >Naam eredienst input</AuLabel>
```

Comment:
I'm a bit confused, why can't we use this instead of the `<p><strong>` (line 176)? Now the text will be read twice to screen readers and the "label" can't be clicked.

Reply:
It's the pattern for the editable cards https://appuniversum.github.io/ember-appuniversum/docs/patterns/au-card-edit

Reply:
I guess that pattern needs to be improved then :smile:.

Reply:
Ok, TODO 😬